### PR TITLE
Resolve #51 Add abbreviations section

### LINKED ISSUE #51: Abbreviations and Definitions Section
Check one:
- [ ] Semantic Issue: something about the meaning of the text is different
- [x] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:
Currently, abbreviations are not used at all in the document, and everything is written out fully, which gets long and annoying to read. Would be useful to have a section that defines common abbreviations for terms so that we can use those throughout the document.

### PR DESCRIPTION
**Pull request by esoccoli**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [X] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Wrote a first draft of an abbreviations section. I intend to bring this up for discussion during tomorrows meeting (9/26/23).

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -581,128 +606,129 @@
...
-            The other Introductory Members will have Off-Floor status, but will otherwise have the same privileges and responsibilities.
+	\item During Spring semester, the Evals Director selects a group of Active Members to form a Selections Committee.
+		The Selections Committee reviews applications and conducts interviews in accordance with the process prescribed by ResLife.
+        \item A subset of applicants will be offered Introductory Membership and must participate in the Intro Process defined in \ref{The Introductory Process}. 
+            A subset of Intro Members will be offered On-Floor status and will be able to select a room on floor. 
+            The other Intro Members will have Off-Floor status, but will otherwise have the same privileges and responsibilities.
 \end{enumerate}
-Note: Most membership privileges do not initiate until successful completion of the introductory process.
-This means that until the member has passed the Introductory Evaluation, the member does NOT have the right to vote on House issues and does NOT count towards quorum.
+Note: Most membership privileges do not initiate until successful completion of the intro process.
```
Comment (rhochgraf21) [inline constitution.tex:641]: capitalize Intro Process?

File: constitution.tex [hunk 2]
```diff
@@ -907,10 +933,10 @@
 \end{enumerate}
 
 \asection{Appeals}
-Decisions made by the Executive Board may be appealed and overturned.
-To initiate an appeal, a member must have the support of three voting members of the Executive Board, or a petition with the signatures of one-third of Active Members.
-After the appeal is presented, a Simple Majority vote is held by the Executive Board whether or not to overturn and reevaluate the decision.
-If the vote passes, the Executive Board may discuss and make a new ruling by another Simple Majority vote.
+Decisions made by E-Board may be appealed and overturned.
+To initiate an appeal, a member must have the support of three voting E-Board members, or a petition with the signatures of one-third of Active Members.
```
Comment (rhochgraf21) [inline constitution.tex:937]: capitalize members

**Review by rhochgraf21 — CHANGES_REQUESTED** [review]
hoping this doesn't give you 18 emails :(
see review notes

**Comment by esoccoli** [comment]
I am going to resolve all the things that I intentionally didn't change, but I am open to discussion

File: constitution.tex [hunk 3]
```diff
@@ -8,6 +8,7 @@
 \documentclass{article}
 \providecommand{\RevisionInfo}{}
 \usepackage{hyperref}
+
```
[the remark this answers is not in the document]
Comment (esoccoli) [reply constitution.tex:11]: Are you referring to the newline? If so i can look at it, but I dont think that is a necessary change for merging this

**Comment by esoccoli** [comment]
Can we push the some of these changes off and put them in the spelling and grammar megathread? I am fine with changing it, but I don't think this PR is the right place because all of my changes keep the same capitalization as the current version

File: constitution.tex [hunk 4]
```diff
@@ -166,31 +190,31 @@
...
-Introductory members are expected to meet all the requirements of the Introductory Process, as described in \ref{Expectations of an Introductory Member}.
+Intro members are expected to meet all the requirements of the Intro Process, as described in \ref{Expectations of an Introductory Member}.
 \asubsection{Introductory Membership Privileges}
-Introductory members receive the right to use Computer Science House facilities, to attend House functions, and to have housing priority over all persons who are not Computer Science House members.
+Intro members receive the right to use Computer Science House facilities, to attend House functions, and to have housing priority over all persons who are not Computer Science House members.
 \asubsection{Introductory Membership Evaluations}
-Introductory members are evaluated on their performance during the introductory period.
-The introductory evaluation process is described in \ref{Introductory Evaluation}.
+Intro members are evaluated on their performance during the intro period.
+The intro evals process is described in \ref{Introductory Evaluation}.
```
[the remark this answers is not in the document]
Comment (esoccoli) [reply constitution.tex:203]: I would say pluralized

**Comment by esoccoli** [comment]
Made all the requested changes, and also added newlines after each section to make it easier to find things in the document. 

These newlines can be removed later, but for working on changes, it is very difficult to read the latex document when there are giant groups of like 20 lines of text with no line break

**Comment by rhochgraf21** [comment]
I think this looks good. This is a large enough commit that a second reviewer would be a good idea. @Sam-Cordry ?

File: constitution.tex (hunk 4 above)
[the remark this answers is not in the document]
Comment (danisaba) [reply constitution.tex]: I feel like it should be plural for mentions where its plural, and singular in cases where its not. I think we already talked about this and got agreement during a meeting earlier this semester regarding this as well

**Review by Sam-Cordry — APPROVED** [review]
Looks good for now; we'll probably and more to this section later as we flesh out what we think should go here

**Review by rhochgraf21 — APPROVED** [review]

**Pull request merged**